README.md

```diff
@@ -76,6 +76,17 @@ MyType.prototype[fl.map] = MyType.prototype.map
 var foo = bar[fl.map](x => x + 1)
 ```
 
+## Type representatives
+
+Certain behaviours are defined from the perspective of a member of a type.
+Other behaviours do not require a member. Thus certain algebras require a
+type to provide a value-level representative (with certain properties). The
+Identity type, for example, could provide `Id` as its type representative:
+`Id :: TypeRep Identity`.
```

Comment:
The part which confused me was `Id :: TypeRep Identity`  (`Id` and `Identity` are different when `Number` and `Number` are same) is it because we assume definition of Identity would be something like this `type Identity a = Id a`?

So if we have:
```hs
type Maybe a = Just a | Nothing
```

Then is this true:?
```hs
Just a :: TypeRep (Maybe a)
Nothing :: TypeRep (Maybe a)
```